thirdparty/README.md

```diff
@@ -263,6 +272,25 @@ changes to ensure they build for Javascript/HTML5. Those
 changes are marked with `// -- GODOT --` comments.
 
 
+## Vulkan Ecosystem Components (Vulkan ICD loader and headers)
```

Comment:
Sections should be named like the folder they refer to (another convention that I'm the only one to understand from the context :D). And this should go further down to respect alphabetical ordering.

Reply:
Fixed by #31102.